pkg/ecctl/init.go

```diff
@@ -67,6 +73,13 @@ const (
 )
 
 var (
+	hostChoiceMsg = `
+Select which type of infrastructure this configuration will for:
+  [1] Elastic Cloud (Elasticsearch Service).
```

Comment:
I'd like @Kushmaro to confirm the wording here. It seems that the official name is `Elasticsearch Service` so imho this should be the name not inside the parenthesis, but I might be wrong

Reply:
thanks @ppapapetrou76 for the mention!
@karencfv , I'd change the wording to - 
"Select which type of Elastic Cloud offering will you be working with:"
[1] Elasticsearch Service (default)
[2] Elastic Cloud Enterprise (ECE) or Elasticsearch Service Private (ESSP)

 if no choice is enter - [1] should be selected.
in case 1 is selected, we could also output something like - 
"Using https://api.elastic-cloud.com as the API host" 

and continue the flow